ios/Framework/Core/Asset/DisplayScreenlet/AssetDisplayBuilder.swift

```diff
@@ -19,16 +19,16 @@ import UIKit
 	open class func createScreenlet(_ frame: CGRect, asset: Asset, themeName: String? = nil)
 			-> BaseScreenlet? {
 
-		if asset.isAnyMimeType(ImageDisplayScreenlet.supportedMimeTypes) {
+		if asset.isAnyMimeType(ImageDisplayScreenlet().supportedMimeTypes) {
 			return ImageDisplayScreenlet(frame: frame, themeName: themeName)
```

Comment:
If you instanciate the class here the user wont be able to override the mimetypes, isn't it?